.github/workflows/node.js.yml

```diff
@@ -28,9 +28,6 @@ jobs:
       - run: npm install -g npm@7 # update to npm 7
```

Comment:
You can install last npm version as you droped nodejs 10.we don't need to pin to 7 anymore

Reply:
ah true